packages/orchestrator-ui-components/src/hooks/useQueryWithGraphql.ts

```diff
@@ -29,5 +22,7 @@ export const useQueryWithGraphql = <U, V extends Variables>(
         // @ts-ignore
         await graphQLClient.request(query, queryVars);
 
-    return useQuery([queryKey, ...Object.values(queryVars)], fetchFromGraphql);
+    return useQuery([queryKey, ...Object.values(queryVars)], fetchFromGraphql, {
+        refetchInterval,
+    });
 };
```

Comment:
Since `useQueryWithGraphql` is just a small wrapper around `useQuery`, you could consider allowing any `useQuery` options and pass them to `useQuery`. That's would be more flexible than only allowing `refetchInterval` by hand.

Reply:
Will implement this when there is a need for the extra props, polling is a temporary solution for now